src/app/shared/interfaces/gagestatistic.ts

```diff
@@ -13,10 +13,13 @@ export interface GageStatistic {
     comments: string;
     isPreferred: boolean;
     yearsofRecord: number;
-    statisticGroupType?: Statisticgroup;
+    //statisticGroupType?: Statisticgroup;
```

Comment:
This should be part of the gage return now!

Reply:
Opps. My bad.